mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_feature_test.go.erb

```diff
@@ -488,6 +488,7 @@ resource "google_gke_hub_feature" "feature" {
   fleet_default_member_config {
     configmanagement {
       version = "1.16.1"
+      prevent_drift = true
       config_sync {
```

Comment:
This should be inside config_sync (which is why you're getting an alert about missing tests).

 ```suggestion
      config_sync {
        prevent_drift = true
```